File: Python/zadanie2.py
print("Zadanie 2")

kierunek = input("Wybierz kierunek (C - z Celsjusza na Fahrenheit, F - z Fahrenheita na Celsjusz): ").upper()

if kierunek == "C":
    celsjusz = float(input("Podaj temperaturę w °C: "))
    fahrenheit = celsjusz * 1.8 + 32
    print(f"{celsjusz}°C = {fahrenheit}°F")
elif kierunek == "F":
    fahrenheit = float(input("Podaj temperaturę w °F: "))
    celsjusz = (fahrenheit - 32) / 1.8
    print(f"{fahrenheit}°F = {celsjusz}°C")
else:
    print("Niepoprawny wybór! Wybierz C lub F.")

File: C#/zadanie2.cs
Console.WriteLine("Zadanie 2");

Console.Write("Wybierz kierunek (C - z Celsjusza na Fahrenheit, F - z Fahrenheita na Celsjusz): ");
string kierunek = Console.ReadLine().ToUpper();

if (kierunek == "C")
{
    Console.Write("Podaj temperaturę w °C: ");
    double celsjusz = Convert.ToDouble(Console.ReadLine());
    double fahrenheit = celsjusz * 1.8 + 32;
    Console.WriteLine(celsjusz + "°C = " + fahrenheit + "°F");
}
else if (kierunek == "F")
{
    Console.Write("Podaj temperaturę w °F: ");
    double fahrenheit = Convert.ToDouble(Console.ReadLine());
    double celsjusz = (fahrenheit - 32) / 1.8;
    Console.WriteLine(fahrenheit + "°F = " + celsjusz + "°C");
}
else
{
    Console.WriteLine("Niepoprawny wybór! Wybierz C lub F.");
}